ydb/tests/functional/tenants/test_db_counters.py

```diff
@@ -129,3 +129,239 @@ def test_case(self):
+
+        assert False, "database did not move into DiskQuotaExceeded state"
+
+    def check_counters(self, mon_port, sensors_to_check, retries, sleep_duration):
+        for attempt in range(retries + 1):
+            counters = get_db_counters(mon_port, "ydb")
+            correct_sensors = 0
+            if counters:
+                for sensor in counters["sensors"]:
+                    for target_name, expected_value in sensors_to_check.items():
+                        if sensor["labels"]["name"] == target_name:
+                            logger.debug(
+                                f"sensor {target_name}: expected {expected_value}, "
+                                f'got {sensor["value"]} in {sleep_duration * attempt} seconds'
+                            )
+                            if sensor["value"] == expected_value:
+                                correct_sensors += 1
+                                if correct_sensors == len(sensors_to_check):
+                                    return
+
+            logger.debug(
+                f"got {correct_sensors} out of {len(sensors_to_check)} correct sensors "
+                f"in {sleep_duration * attempt} seconds"
+            )
+            time.sleep(sleep_duration)
+
+        assert False, (
+            f"didn't receive expected values for sensors {sensors_to_check.keys()} "
+            f"in {sleep_duration * retries} seconds"
+        )
+
+
+class TestStorageCounters(StorageCounters):
+    def test_storage_counters(self):
+        self.alter_database_quotas(
+            self.database,
+            """
+            storage_quotas {
+                unit_kind: "hdd"
+                data_size_hard_quota: 2
+                data_size_soft_quota: 1
+            }
+            storage_quotas {
+                unit_kind: "hdd1"
+                data_size_hard_quota: 20
+                data_size_soft_quota: 10
+            }
+            """,
+        )
+
+        quotas = self.describe(self.database).PathDescription.DomainDescription.DatabaseQuotas.storage_quotas
+        assert len(quotas) == 2
+        assert json_format.MessageToDict(quotas[0], preserving_proto_field_name=True) == {
+            "unit_kind": "hdd",
+            "data_size_hard_quota": "2",
+            "data_size_soft_quota": "1",
+        }
+        assert json_format.MessageToDict(quotas[1], preserving_proto_field_name=True) == {
+            "unit_kind": "hdd1",
+            "data_size_hard_quota": "20",
+            "data_size_soft_quota": "10",
+        }
+
+        slot_mon_port = self.cluster.slots[1].mon_port
+        # Note 1: limit_bytes is equal to the database's SOFT quota
+        # Note 2: .hdd counter aggregates quotas across all storage pool kinds with prefix "hdd", i.e. "hdd" and "hdd1"
+        # Note 3: 200 seconds can sometimes be not enough
+        self.check_counters(slot_mon_port, {"resources.storage.limit_bytes.hdd": 11}, retries=60, sleep_duration=5)
```

Comment:
Please make sure you are ok with this

Reply:
Also, it is important to note that the wait time must be at least 200 seconds according to my tests on a heavily loaded local machine. It is already equal to [150 seconds](https://github.com/ydb-platform/ydb/blob/f835e78f443414d6158c60b63c9e790e763b55ec/ydb/tests/functional/tenants/test_db_counters.py#L119) for the other (previously existing) test in this file.

Reply:
> Please make sure you are ok with the reported value being 11, i.e. a sum of the soft quotas of "hdd" and "hdd1" storage pools.

That is ok for now.